net/http/civetweb/civetweb.c

```diff
@@ -19046,9 +19046,11 @@ mg_get_system_info_impl(char *buffer, int buflen)
 	/* Build date */
 	{
 #if defined(GCC_DIAGNOSTIC)
+#if GCC_VERSION >= 50000
```

Comment:
My GCC doesn't have `GCC_VERSION`?! `echo | gcc -dM -E -x c++ -| grep GCC_VERSION`

Reply:
> My GCC doesn't have GCC_VERSION

It is defined inside civetweb

Reply:
Cool. I'm jealous.